docs/reference/modules/ROOT/pages/lucene.adoc

```diff
@@ -0,0 +1,78 @@
+= Lucene
+:toc: macro
+
+toc::[]
+
+Lucene module for Crux making use of https://lucene.apache.org/[Apache Lucene].
+
+== Setup
+
+`crux-lucene` runs in-process as part of the Crux node as a Crux module.
+
+First, add the `crux-lucene` dependency to your project:
+
+.pom.xml
+[source,xml]
+----
+<dependency>
+    <groupId>juxt</groupId>
+    <artifactId>crux-lucene</artifactId>
+    <version>20.09-1.11.0-alpha</version>
+</dependency>
+----
+
+.deps.edn
+[source,clojure]
+----
+juxt/crux-lucene {:mvn/version "20.09-1.11.0-alpha"}
+----
+
+Add the following to your node configuration:
+
+.JSON
+[source,json]
+----
+{
+  "crux.lucene/node": {
+        "db-dir": "lucene",
+        "crux/indexer": {
+            "crux/module": "crux.lucene/->indexer",
+            "indexer": "crux.kv.indexer/->kv-indexer"
+        }
+  }
+}
+----
+
+.Clojure/EDN
+[source,clojure]
+----
+{...
+ :crux.lucene/node {:db-dir "lucene-dir"
+                    :crux/indexer {:crux/module 'crux.lucene/->indexer
+                                   :indexer 'crux.kv.indexer/->kv-indexer}}
+----
+
+
+== Querying
```

Comment:
should we mention the four returned attributes in the tuple?

Reply:
Done.